src/parser/scanner.lex

```diff
@@ -1,3 +1,5 @@
+/* Flex definitions section */
```

Comment:
Why we had different comment style in this pr?

Reply:
These are some of the limitations of flex itself. 
The comment style `//` is forbidden by flex in the definition section of file scanner.lex.